packages/rosaenlg-doc/doc/modules/mixins_ref/pages/verbs_french.adoc

```diff
@@ -38,6 +50,8 @@ If the auxiliary is not set, these rules will apply:
 
 TIP: `PASSE_COMPOSE` can be used to mimic the passive form. `elle &#35;[+verb(getAnonFS(), {verb: 'convoquer', tense:'PASSE_COMPOSE', aux:'ETRE', agree:getAnonFS()})]` will output _elle est convoquée_. But `elle &#35;[+subjectVerbAdj(getAnonFS(), 'être', 'convoqué')]` generates the same output and is much simplier.
 
+You can also use the parameter `voice: 'Pass'`

```

Comment:
Can you please add an example using Pass voice in the main doc, like for pronominal form just below? It can be the same as in the unit tests.

Reply:
done